src/panels/lovelace/sections/hui-section.ts

```diff
@@ -131,6 +124,14 @@ export class HuiSection extends ReactiveElement {
       }
       if (changedProperties.has("lovelace")) {
         this._layoutElement.lovelace = this.lovelace;
+        this._cards.forEach((element) => {
+          try {
```

Comment:
This try catch should probably be moved to `hui-card`, same for the try catch when setting `hass` on line 118.

Reply:
I will improve that in a next PR 👍